rag/easy-rag/deployment/pom.xml

```diff
@@ -35,15 +35,9 @@
         </dependency>
         <dependency>
             <groupId>dev.langchain4j</groupId>
-            <artifactId>langchain4j-embeddings-all-minilm-l6-v2-q</artifactId>
+            <artifactId>langchain4j-embeddings-bge-small-en-v15-q</artifactId>
```

Comment:
Why is this needed in `deployment`? Easy RAG shouldn't depend on any particular embedding model.

The documentation also says

> After adding this extension to your application, all that is needed to ingest documents into an embedding store is to add a dependency that provides an embedding model

Reply:
It's a test dependency. 
You need an embedding model in your tests.

Also, upstream easy RAG depends on it (which turned to be a bit of a nightmare).

Reply:
Ah, right, I didn't notice the scope.

Ok, so, another idea: I think we could switch from depending on  `dev.langchain4j:angchain4j-easy-rag` to just `dev.langchain4j:langchain4j-document-parser-apache-tika`?  I don't think we actually use anything from the Easy RAG module so I suppose it should still work. The upstream Easy RAG is different from ours. Then you also won't have the dependency on BGE.

Reply:
It won't work; you are still using the HuggingFaceTokenizer in your code.